aiohttp_security/api.py

```diff
@@ -48,6 +89,16 @@ def forget(request, response):
 
 
 @asyncio.coroutine
+def get_user_identity(request):
```

Comment:
`authorized_userid` should be used instead of `get_user_identity`.
I see no reasons for `get_user_identity` existence.